api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoaderTest.kt

```diff
@@ -49,31 +49,22 @@ class ModelLoaderTest {
             }
 
             with((this[7] as Dense).kernelInitializer) {
```

Comment:
Not related to your changes but this method name does not follow our [contribution guideline](https://github.com/Kotlin/kotlindl/blob/master/CONTRIBUTING.md#submitting-prs)
```
Follow the style of writing tests that is used in this project: name test classes as xxxTest. Don't use backticks or underscores in test names.
```